domain/consensus/processes/consensusstatemanager/resolve_block_status.go

```diff
@@ -78,14 +89,32 @@ func (csm *consensusStateManager) resolveBlockStatus(stagingArea *model.StagingA
 				return 0, err
 			}
 		}
+		if blockStatus != externalapi.StatusUTXOValid {
```

Comment:
Why do we break? Don't we need to mark the rest of the blocks as disqualified?

Reply:
Good catch! Not sure what I thought back then